database/schema/sqlite-schema.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE IF NOT EXISTS "migrations" ("id" integer primary key autoincrement not null, "migration" varchar not null, "batch" integer not null);
 CREATE TABLE IF NOT EXISTS "Console" ("ID" integer primary key autoincrement not null, "Name" varchar not null, "Created" datetime, "Updated" datetime, "name_full" varchar, "name_short" varchar, "manufacturer" varchar, "active" tinyint(1), "order_column" integer, "deleted_at" datetime);
-CREATE TABLE IF NOT EXISTS "EmailConfirmations" ("User" varchar not null, "EmailCookie" varchar not null, "Expires" date not null);
+CREATE TABLE IF NOT EXISTS "EmailConfirmations" ("id" integer primary key autoincrement not null, "User" varchar not null, "EmailCookie" varchar not null, "Expires" date not null);
```

Comment:
Was [this change](https://github.com/RetroAchievements/RAWeb/pull/2598/commits/aa4375b327085d0a78bf11b09cd1a47376e899c5#diff-3c4872576f46a8f07b1ada7f70f48973ed38eb1730c678c98ad16f5bf141288eR3) intentional (addition of ID column)? It doesn't appear in the mysql-schema file.